agave_app/citationDialog.cpp

```diff
@@ -40,14 +43,16 @@ CitationDialog::CitationDialog()
   citationtext2->setText("<tt>@software{agave,<br/>"
                          "&nbsp;&nbsp;author    = {Toloudis, Daniel and AGAVE Contributors},<br/>"
                          "&nbsp;&nbsp;title     = {AGAVE: Advanced GPU Accelerated Volume Explorer},<br/>"
-                         "&nbsp;&nbsp;year      = {2023},<br/>"
+                         "&nbsp;&nbsp;year      = {" +
+                         syear +
+                         "},<br/>"
                          "&nbsp;&nbsp;version = {" +
                          qApp->applicationVersion() +
                          "},<br/>"
                          "&nbsp;&nbsp;url       = {" +
                          agaveUrl +
-                         "}<br/>"
-                         "&nbsp;&nbsp;organization = {Allen Institute for Cell Science}<br/>"
+                         "},<br/>"
+                         "&nbsp;&nbsp;organization = {Allen Institute for Cell Science},<br/>"
                          "&nbsp;&nbsp;note      = {Computer Software}<br/>"
```

Comment:
do you need a comma after Software} here?

Reply:
nope! `note` is the last one in the list